cartoframes/data/__init__.py

```diff
@@ -1,7 +1,11 @@
+from __future__ import absolute_import
+
 from .dataset.dataset import Dataset
 from .dataset.registry.strategies_registry import StrategiesRegistry
+from .clients.sql_client import SQLClient
 
 __all__ = [
     'Dataset',
-    'StrategiesRegistry'
+    'StrategiesRegistry',
+    'SQLClient'
```

Comment:
Why is this exposed in `cartoframes.data` now (insdead of `cartoframes.data.clients`)?

Reply:
We should be consistent with the current API, so let's move it back to data.clients